lib/rucio/web/ui/common/utils.py

```diff
@@ -73,6 +75,7 @@
     except:
         AUTH_ISSUERS = []
 
+MULTI_VO = config_get('common', 'multi_vo', default=False)
```

Comment:
please use `config_get_bool` here. Otherwise multi_vo will always be enabled as soon it's added to the config file, even for `multi_vo = False`

Reply:
Done.